### ISSUE TITLE
[Suggestion] Toggle walk/run

### ISSUE THREAD

**Issue by nashmuhandes** [Interkarma/daggerfall-unity#1389]
A feature found in most first person games that lets you toggle walking or running, so that you don't have to hold down the Shift button. Usually bounded to Caps Lock by default in most the games that have this feature. :)

**Comment by petchema** [comment]
It's littterally already on the roadmap for 0.10.x:

https://www.dfworkshop.net/projects/daggerfall-unity/roadmap/

**Comment by l0b0** [comment]
I can't seem to find this feature in 1.1.1. It looks like _autorun_ was implemented instead of _toggle run_.

**Comment by petchema** [comment]
Jay_H and I described how it works in the forums
https://forums.dfworkshop.net/viewtopic.php?p=68391#p68391

**Comment by KABoissonneault** [comment]
It seems that the functionality is not what people expect. 

For the scenario of going forward while running with a single held key, the approach you describe is:
- Toggle "auto forward" 
- Hold Run to make the character automatically run forward

The requested feature is:
- Toggle "auto run"
- Hold forward to make the character manually run forward

I can see how the first approach is not great for multiple short bursts of movement, and more suited for long travel. I think the requested feature makes sense, even if we might have to give it another name compared to what "auto run" means for us currently

**Comment by nashmuhandes** [comment]
Yes, the requested feature is how it is commonly in most first person shooters like Doom or Quake, or actually even in Morrowind, Oblivion and Skyrim. The Caps Lock key is typically used to toggle "always run" on or off.

The requested feature in the OP is **NOT** "automatic forward" where the player could let go of the controls while the avatar moves forward with no player input.
